common/beerocks/bwl/CMakeLists.txt

```diff
@@ -27,7 +27,12 @@ elseif(TARGET_PLATFORM STREQUAL "openwrt")
     if (TARGET_PLATFORM_TYPE STREQUAL "ugw")
         set(BWL_TYPE "DWPAL")
     else()
-        set(BWL_TYPE "NL80211")
+        # For gneric openwrt target use NL80211 interface by default
+        if (NOT DEFINED ENV{PRPLMESH_BWL_TYPE})
+            set(BWL_TYPE "NL80211")
+        else()
+            set(BWL_TYPE "$ENV{PRPLMESH_BWL_TYPE}")
+        endif()
```

Comment:
Here, I think the existing code already works, so this change is not needed.

Long term, we want BWL_TYPE to be a cache variable and only set it based on PLATFORM if it is not defined already. Longer term, we'll want to remove PLATFORM entirely.

Reply:
Ok, will remove